docs/modules/reference/pages/service-assurance/monitors/DeviceConfigMonitor.adoc

```diff
@@ -34,17 +34,20 @@ Don't include `.dcb` extension when specifying the file name in metadata.
 | username
 | Username for SSH login. 
 Overwrite in metadata as `dcb:username`.
+Note that you can use Secure Credentials Vault (SCV) metadata.
```

Comment:
Make "Secure Credentials Vault (SCV) metadata" a link to the documentation on SCV metadata.

Reply:
Done